index.html

```diff
@@ -4,36 +4,173 @@
     <title>My portfolio</title>
 </head>
 <body>
     
-    <div class="wrapper">
-        
-       <header>
-            <nav class="nav-wrapper">
+    <section class="wrapper-header">       
+     <header>
+         <nav class="nav-wrapper">
             
-                <a href="#" class="link-hamburger-icon">☰</a>
-                <!-- <ul>
-                    <li><a href="#">Hello</a></li>
-                    <li><a href="#">Portfolio</a></li>
-                    <li><a href="#">About</a></li>
-                    <li><a href="#">Contact</a></li>
+            <a href="#" class="link-hamburger-icon">☰</a>
+            <!-- <ul>
+                <li><a href="#">Hello</a></li>
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>                    <li><a href="#">Contact</a></li>
                 </ul> -->
-            </nav> 
+         </nav> 
 
 
-            <section class="headline">
-                <h1 class="name-headline">Charlie<br>Giorgania</h1>
-                <p class="content-headline">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
-                    If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
-            </section>
+        <div class="headline">
+            <h1 class="name-headline">Charlie<br>Giorgania</h1>
+            <p class="content-headline">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+                If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+        </div>
 
-            <section class="collaboration-headline">
-                <p>Start collaboration</p>
-            </section>
-        </header>
-    </div>
+        <section class="collaboration-headline">
+            <p>Start collaboration</p>
+        </section>
+     </header>
+
+        <section class="separation-headline-projects"> </section>
+    </section>
+    <section class="wrapper-projects-section">
+        <div class="title-project-section">
+            <h1>Projects</h1>
+        </div>
+        <div class="wrapper-projects">
+            <div class="projects-index project1-section">
+                <img src="/04_Mockups/project1-mockup.png" alt="Project 1 Mockup" class="project-mockup project1-mockup">
+                <h2 class="projectname projectname1">
+                    Project name goes here
+                </h2>
+                <ul class="projects-languages project-languages1">
+                    <li>HTML/CSS</li>
+                    <li>Ruby on Rails</li>
+                    <li>JavaScript</li>
+                </ul>
+                <div class="see-project-btn">
+                    <p>See this project <img src="/03_Resources/01_Graphics/02_Stock/right-white-arrow.svg" alt="Right white arrow" class="arrow-see-project-btn"></p>
```

Comment:
- [ ] Remember this is a link. Let's try to add `a` tag to make our project professional. Kindly do it for all the projects 😃